engine/src/main/resources/assets/i18n/menu_id.lang

```diff
@@ -0,0 +1,255 @@
+    "camera-blur-max": "Maks",
+    "camera-blur-normal": "Normal",
+    "camera-blur-off": "Mati",
+    "camera-blur-some": "Sebagian",
+    "camera-details-settings": "Kamera / Pengaturan Detil Grafik",
+    "camera-film-grain": "Film Grain",
+    "camera-setting": "Pengaturan Kamera",
+    "camera-setting-cinematic": "Sinematic",
+    "camera-setting-normal": "Normal",
+    "camera-setting-smooth": "Halus",
+    "category-behavior": "Sifat",
+    "category-general": "Umum",
+    "category-interaction": "Interaksi",
+    "category-inventory": "Inventaris",
+    "category-mouse": "Mouse",
+    "category-movement": "Gerakan",
+    "category-nui": "NUI",
+    "change-keybind-popup-message": "sudah terikat kepada suatu tindak. Tetap ikat ulang?",
+    "change-keybind-popup-title": "Tombol sudah terikat",
+    "clamp-lighting": "Pencahayaan Clamp",
+    "cloud-shadows": "Bayangan Awan",
+    "connecting-to": "Menghubungkan ke",
+    "connection-failed": "Koneksi gagal!",
+    "could-not-connect-to-server": "Tidak bisa menghubungkan ke server",
+    "credits": "Kredit",
+    "create-game": "Buat",
+    "create-game-title": "Buat Permainan",
+    "create-preview": "Pratinjau",
+    "custom-servers": "Custom",
+    "deactivate-module": "Nonaktifkan",
+    "death-screen-title": "Anda mati",
+    "default-binding": "Ikatan semula",
+    "delete-game": "Hapus",
+    "dev-tools-bt-editor": "Editor Pohon Sifat",
+    "dev-tools-nui-editor": "Editor Layar NUI",
+    "dev-tools-skin-editor": "Editor Skin NUI",
+    "dev-tools-title": "Alat Deveoper",
+    "dialog-cancel": "Batal",
+    "dialog-ok": "Ok",
+    "disable-all-modules": "Nonaktifkan Semua",
+    "download-module": "Unduh",
+    "downloading-server-list": "Mengundah daftar server ..",
+    "drop-item": "Drop Barang",
+    "edit-server": "Edit",
+    "edit-server-title": "Tambahkan/Edit Server",
+    "enter-username-message": "Halo dan selamat datang ke multiplayer Terasology's !\nApakah datang ingin menggunakan username lain?",
+    "enum-editor-prompt": "Tentukan nilai untuk node:",
+    "enum-editor-title": "Editor Enum Node",
+    "error-downloading-server-list": "Error sangat mengundah daftar server!",
+    "exit-game": "Keluar dari Terasology",
+    "extra-lighting": "Pencahayaan Ekstra",
+    "eye-adaption": "Adaptasi mata",
+    "failed-to-join": "Gagal untuk masuk",
+    "flickering-lights": "Lampu Berkedip",
+    "fog": "Kabut",
+    "fov-amount": "Banyak FoV",
+    "framerate-limit": "Limit framerate",
+    "game-name": "Nama Permainan",
+    "game-settings": "Pengaturan",
+    "game-worlds": "Dunia",
+    "gameplay": "Template Dunia",
+    "gameplay-info": "bisa  meng-klik 'modul' untuk menyesuaikan lebih lanjut.",
+    "host-multiplayer-game": "Host Permainan",
+    "ingame-menu-title": "menu Permainan",
+    "input-settings": "Input",
+    "input-settings-title": "Pengaturan Input",
+    "inspection-set": "Setel",
+    "invert-mouse": "Balikan Kontrol Mouse",
```

Comment:
Mouse as in computer hardware does not have a translation in ID. Only the one for animal has.